src/docs/style/logo.mdx

```diff
@@ -0,0 +1,267 @@
+
+## Download the App Logo Template
+
+-   Adobe Illustrator
+    -   [iOS](https://github.com/brightlayer-ui/doc-it/blob/master/src/app/assets/styles/logos/templates/iOS-IllustratorTemplate.ai?raw=true)
+    -   [Android Adaptive Icon](https://github.com/brightlayer-ui/doc-it/blob/master/src/app/assets/styles/logos/templates/Android-Adaptive-Icon-IllustratorTemplate.ai?raw=true)
+-   [PDF (for printing)](https://raw.githubusercontent.com/brightlayer-ui/doc-it/master/src/app/assets/styles/logos/App%20Logos%20Printable.pdf)
+-   [SVG](https://github.com/brightlayer-ui/doc-it/tree/master/src/app/assets/styles/logos/templates/SVG)
+-   Our [Figma Component Sticker Sheet](https://www.figma.com/community/file/1024360297793425107) also includes a component called "App Logo Template"
+
+## Glyph
+
+A glyph is a simple shape representing an object, action, or idea. Your glyph must be simple, recognizable, and scalable to ensure that your app can be easily identified on multiple devices with different resolutions.
+
+We recommend you select your base glyph from our [icon library](/style/icon-library).
+
+<ImageGrid
+    images={[Glyph1, Glyph2]}
+    captionsUnderImages={[
+        `AVOID: Avoid the use of mnemonic logos or logotypes when possible. They are often not meaningful for first-time users, and can be hard to see in small logo sizes.`,
+        `DO: Use a graphical way to represent an application or product.`,
+    ]}
+/>
+
+Do not use the Eaton logo as a glyph — your app should not be presented as the "Eaton" app. It is not unusual for a customer to have multiple Eaton products and applications installed on their devices, and if every one of those apps used the Eaton logo it becomes very difficult for a user to tell, at a glance, which application they are looking for.
+
+<ImageGrid images={[Glyph3]} captionsUnderImages={[`DONT: Do not use the Eaton logo as glyphs.`]} />
+
+You should also verify that your product logo is distinguished enough from other existing Eaton product logos.
+
+Glyphs are usually aligned to the keyline (visible in the template) to avoid bleeding beyond the safe zone.
+
+<ImageGrid images={[GlyphKeyline]} captionsUnderImages={[`Align glyphs to the icon template keyline.`]} regularWidth />
+
+Once the base glyph is decided, you can be creative and add variations to the glyph to make it look more designed.
+
+<ImageGrid
+    images={[PolishGlyph1, PolishGlyph2]}
+    captionsUnderImages={[
+        `Smart Energy Manager modified the glyph shape to accommodate for a bolt glyph inside.`,
+        `Predict Pulse modified the glyph size, opacity, and image gradient.`,
```

Comment:
Just a small point...since we moved the glyph section to be first, talking about the customization of the background is a tiny bit premature at this point in the doc...I think it's fine, just pointing it out.